src/latch/ldata/_transfer/download.py

```diff
@@ -72,10 +74,34 @@ def download(
     else:
         endpoint = latch_config.api.data.get_signed_url
 
+    egress_source: Optional[dict[str, str]] = None
+
+    try:
+        pod_id = Path("/root/.latch/id").read_text("utf-8")
+        egress_source = {"pod_id": pod_id}
+    except FileNotFoundError:
+        sdk_token = user_config.token
+        if sdk_token != "":
+            egress_source = {"sdk_token": sdk_token}
```

Comment:
Can we just indicate an SDK token? IDK if we want to shit the token into the event logs.